src/components/data_comps/dlnd/cime_config/config_component.xml

```diff
@@ -76,7 +98,7 @@
 
   <entry id="DLND_CPLHIST_YR_ALIGN">
     <type>integer</type>
-    <default_value>1</default_value>
+    <default_value>-999</default_value>
```

Comment:
See above comment for DATM_CPLHIST_YR_ALIGN: whatever documentation is added for that one should also be added for DLND_CPLHIST_YR_ALIGN